resources/views/partials/content-front-page.blade.php

```diff
@@ -0,0 +1,46 @@
+<div class="home__search">
+  @include('partials.search-form', ['modifier' => 'inverse', 'placeholder' => __('Search resource name, publisher, or topic…')])
+  {{-- TODO: Add saved searches --}}
+</div>
+<div class="home__browse">
+  <h2>{{ __('Browse by…', 'coop-library') }}</h2>
+  <ul class="link-list link-list--inverse">
+    @foreach([
+    'lc_topic' => __('Topics', 'coop-library'),
+    'lc_goal' => __('Goals', 'coop-library'),
+    'lc_sector' => __('Sectors', 'coop-library'),
+    'lc_format' => __('Formats', 'coop-library'),
+  ] as $slug => $label)
+    <li class="link-list__item"><a href="{{ App::termListUrl($slug) }}">@svg(str_replace('lc_', '', $slug), 'icon--' . str_replace('lc_', '', $slug), ['focusable' => 'false', 'aria-hidden' => 'true']) {{ $label }}</a></li>
+  @endforeach
+  </ul>
+  <div class="wp-block-button wp-block-button--inverse"><a href="{{ get_post_type_archive_link('lc_resource') }}" class="wp-block-button__link">{{ __('Browse all resources', 'coop-library') }}</a></div>
+</div>
+<div class="home__feed">
```

Comment:
Use more semantic html containers (with appropriate heading landmarks) for structuring the page content. Example: Search and Browse can go into a section, and My Feed into another section.